optimum/habana/transformers/models/gptj/modeling_gptj.py

```diff
@@ -1,20 +1,116 @@
 from typing import Optional, Tuple, Union
 
+import habana_frameworks.torch.core as htcore
 import torch
 from torch import nn
 from torch.nn import CrossEntropyLoss
 from transformers.modeling_outputs import BaseModelOutputWithPast, CausalLMOutputWithPast
+from transformers.models.gptj.configuration_gptj import GPTJConfig
 from transformers.models.gptj.modeling_gptj import (
     GPTJMLP,
     GPTJAttention,
+    GPTJBlock,
     GPTJForCausalLM,
+    GPTJModel,
     apply_rotary_pos_emb,
     create_sinusoidal_positions,
     logger,
 )
 
 
+class Matmul(nn.Module):
```

Comment:
@atakaha the `nn.Module` wrapper for FP8 is no longer needed, use the `torch.matmul` directly in the code. [torch.matmul and torch.nn.functional.softmax are supported and replaced by HQT during quantization.](https://docs.habana.ai/en/latest/PyTorch/Inference_on_PyTorch/Inference_Using_FP8.html?highlight=fp8%20#supported-functions)